docs/cn.md

```diff
@@ -423,19 +422,19 @@ container.pagination('go', 8, function(data, pagination){
 ### destroy
 销毁分页实例
 	
-### getSelectedPageNum <em>number</em>
```

Comment:
@superRaytin What's the reason for the name change here? we can enforce code version or make changes to code but changing name might impact users already using it right.  Should this be a major version increment instead of minor version 

In package.json we had ^2.1.5. Due to this code is breaking as the name got updated now in 2.3.0. we are using like this before. 
pagination('**getSelectedPageNum**');

Reply:
@ravitejagundagithub I'm sorry for the trouble caused by the name change, has the problem been solved? The name change is just to improve the syntax. Please let me know if you'd like help, I'll release a patch version that backwards compatible with this name change.

Reply:
If you can release a patch version with backwards compatible that would be great. Instead of using latest version we are using 2.1.5 for now.

Reply:
Try `2.4.2`